jans-cedarling/scripts/parse_binding_benchmarks.py

```
@@ -184,8 +184,8 @@ def render_jsonl_pivot(rows: list[dict]) -> str:
     bindings = sorted(bindings_set)
     scenarios = sorted(scenarios_set)
 
-    out: list[str] = ["### Cross-Platform Binding Benchmarks\n\n"]
-    out.append("#### Mean (µs) per scenario × binding\n\n")
+    out: list[str] = ["## Cross-Platform Binding Benchmarks\n\n"]
+    out.append("### Mean (µs) per scenario × binding\n\n")
```

Ruff (0.16.1)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)